pkg/types/validators.go

```diff
@@ -25,6 +25,21 @@ func (v Validator) WithRunner(fn ValidateFunc) Validator {
 	return v
 }
 
+// ValidatorList - implements Sort interface to sort validators per Code
```

Comment:
```suggestion
var _ sort.Interface = ValidatorList{}

```

Missed this the first time. Better to let the compiler know as well.

As a side note I like putting these in tests via [testify](https://pkg.go.dev/github.com/stretchr/testify@v1.7.0/require#Implements) instead of having weird blank assignments everywhere, but either works well.

Reply:
fixed